doc/using_pytorch.rst

```diff
@@ -566,11 +570,91 @@ The function should return a byte array of data serialized to content_type.
 The default implementation expects ``prediction`` to be a torch.Tensor and can serialize the result to JSON, CSV, or NPY.
 It accepts response content types of "application/json", "text/csv", and "application/x-npy".
 
-Working with Existing Model Data and Training Jobs
-==================================================
 
-Attach to existing training jobs
---------------------------------
+Bring your own model
+====================
+
+You can deploy a PyTorch model that you trained outside of SageMaker by using the ``PyTorchModel`` class.
+Typically, you save a PyTorch model as a file with extension ``.pt`` or ``.pth``.
+To do this, you need to:
+
+* Write an inference script.
+* Package the model artifacts into a tar.gz file.
+* Upload the tar.gz file to an S3 bucket.
+* Create the ``PyTorchModel`` object.
+
+Write an inference script
+-------------------------
+
+You must create an inference script that implements (at least) the ``predict_fn`` function that calls the loaded model to get a prediction.
+Optionally, you can also implement ``input_fn`` and ``output_fn`` to process input and output. 
+For information about how to write an inference script, see `Serve a PyTorch Model <#serve-a-pytorch-model>`_.
+Save the inference script as ``inference.py`` in the same folder where you saved your PyTorch model. 
+
+Package model artifacts into a tar.gz file
+------------------------------------------
+
+The directory structure where you saved your PyTorch model should look something like the following:
```

Comment:
this is true only for more recent versions of PyTorch. also, I recently added a section about the model structure - https://github.com/aws/sagemaker-python-sdk/blob/31c8ff1ad5d1f4377d32262cfd1a378a3a1be71d/doc/using_pytorch.rst#model-directory-structure; we should probably consolidate these?